pages/index.tsx

```diff
@@ -271,6 +271,15 @@ function InstallSection() {
       />
     </div>
   );
+  const pacman = (
+    <div key="pacman" class="my-4 text-gray-700">
+      <p class="mb-2">Pacman (Arch Linux and Arch Based distros):</p>
+      <CodeBlock
+        language="bash"
+        code="sudo pacman -S deno"
```

Comment:
```suggestion
        code="pacman -S deno"
```

No need to interfere with `sudo`.